/work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:283: FutureWarning: `torch.cuda.amp.autocast(args...)` is deprecated. Please use `torch.amp.autocast('cuda', args...)` instead.
  @autocast(enabled=False)
/work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:418: FutureWarning: `torch.cuda.amp.autocast(args...)` is deprecated. Please use `torch.amp.autocast('cuda', args...)` instead.
  @autocast(enabled=False)
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/torch/_dynamo/variables/functions.py:663: UserWarning: Graph break due to unsupported builtin torch._C._to_dlpack. This function is either a Python builtin (e.g. _warnings.warn) or a third-party C/C++ Python extension (perhaps created with pybind). If it is a Python builtin, please file an issue on GitHub so the PyTorch team can add support for it and see the next case for a workaround. If it is a third-party C/C++ Python extension, please either wrap it into a PyTorch-understood custom operator (see https://pytorch.org/tutorials/advanced/custom_ops_landing_page.html for more details) or, if it is traceable, use torch.compiler.allow_in_graph.
  torch._dynamo.utils.warn_once(msg)
V0421 03:41:54.932395 47307089929472 torch/_dynamo/guards.py:2611] [4/1] [__recompiles] Recompiling function create_unitgraph_from_coo in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1254
V0421 03:41:54.932395 47307089929472 torch/_dynamo/guards.py:2611] [4/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:41:54.932395 47307089929472 torch/_dynamo/guards.py:2611] [4/1] [__recompiles]     - tensor 'L['row']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 03:41:54.950613 47307089929472 torch/_dynamo/guards.py:2611] [5/1] [__recompiles] Recompiling function to_dgl_nd in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/__init__.py:141
V0421 03:41:54.950613 47307089929472 torch/_dynamo/guards.py:2611] [5/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:41:54.950613 47307089929472 torch/_dynamo/guards.py:2611] [5/1] [__recompiles]     - tensor 'L['data']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 03:41:54.960711 47307089929472 torch/_dynamo/guards.py:2611] [6/1] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:432
V0421 03:41:54.960711 47307089929472 torch/_dynamo/guards.py:2611] [6/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:41:54.960711 47307089929472 torch/_dynamo/guards.py:2611] [6/1] [__recompiles]     - tensor 'L['data']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 03:41:54.994627 47307089929472 torch/_dynamo/guards.py:2611] [8/1] [__recompiles] Recompiling function torch_dynamo_resume_in_create_unitgraph_from_coo_at_1296 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1296
V0421 03:41:54.994627 47307089929472 torch/_dynamo/guards.py:2611] [8/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:41:54.994627 47307089929472 torch/_dynamo/guards.py:2611] [8/1] [__recompiles]     - tensor 'L['col']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
W0421 03:42:02.582720 47307089929472 torch/_dynamo/variables/tensor.py:715] [30/0] Graph break from `Tensor.item()`, consider setting:
W0421 03:42:02.582720 47307089929472 torch/_dynamo/variables/tensor.py:715] [30/0]     torch._dynamo.config.capture_scalar_outputs = True
W0421 03:42:02.582720 47307089929472 torch/_dynamo/variables/tensor.py:715] [30/0] or:
W0421 03:42:02.582720 47307089929472 torch/_dynamo/variables/tensor.py:715] [30/0]     env TORCHDYNAMO_CAPTURE_SCALAR_OUTPUTS=1
W0421 03:42:02.582720 47307089929472 torch/_dynamo/variables/tensor.py:715] [30/0] to include these operations in the captured graph.
W0421 03:42:02.582720 47307089929472 torch/_dynamo/variables/tensor.py:715] [30/0] 
V0421 03:42:05.546215 47307089929472 torch/_dynamo/guards.py:2611] [4/2] [__recompiles] Recompiling function create_unitgraph_from_coo in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1254
V0421 03:42:05.546215 47307089929472 torch/_dynamo/guards.py:2611] [4/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:05.546215 47307089929472 torch/_dynamo/guards.py:2611] [4/2] [__recompiles]     - tensor 'L['row']' size mismatch at index 0. expected 0, actual 4942
V0421 03:42:05.546215 47307089929472 torch/_dynamo/guards.py:2611] [4/2] [__recompiles]     - tensor 'L['row']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 03:42:05.573833 47307089929472 torch/_dynamo/guards.py:2611] [5/2] [__recompiles] Recompiling function to_dgl_nd in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/__init__.py:141
V0421 03:42:05.573833 47307089929472 torch/_dynamo/guards.py:2611] [5/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:05.573833 47307089929472 torch/_dynamo/guards.py:2611] [5/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 0, actual 4942
V0421 03:42:05.573833 47307089929472 torch/_dynamo/guards.py:2611] [5/2] [__recompiles]     - tensor 'L['data']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 03:42:05.586255 47307089929472 torch/_dynamo/guards.py:2611] [6/2] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:432
V0421 03:42:05.586255 47307089929472 torch/_dynamo/guards.py:2611] [6/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:05.586255 47307089929472 torch/_dynamo/guards.py:2611] [6/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 0, actual 4942
V0421 03:42:05.586255 47307089929472 torch/_dynamo/guards.py:2611] [6/2] [__recompiles]     - tensor 'L['data']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 03:42:05.624096 47307089929472 torch/_dynamo/guards.py:2611] [8/2] [__recompiles] Recompiling function torch_dynamo_resume_in_create_unitgraph_from_coo_at_1296 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1296
V0421 03:42:05.624096 47307089929472 torch/_dynamo/guards.py:2611] [8/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:05.624096 47307089929472 torch/_dynamo/guards.py:2611] [8/2] [__recompiles]     - tensor 'L['col']' size mismatch at index 0. expected 0, actual 4942
V0421 03:42:05.624096 47307089929472 torch/_dynamo/guards.py:2611] [8/2] [__recompiles]     - tensor 'L['col']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 03:42:05.640474 47307089929472 torch/_dynamo/guards.py:2611] [32/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_574 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:574
V0421 03:42:05.640474 47307089929472 torch/_dynamo/guards.py:2611] [32/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:05.640474 47307089929472 torch/_dynamo/guards.py:2611] [32/1] [__recompiles]     - L['___stack0'] == 3459                                      
V0421 03:42:05.853999 47307089929472 torch/_dynamo/guards.py:2611] [34/1] [__recompiles] Recompiling function cat in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:222
V0421 03:42:05.853999 47307089929472 torch/_dynamo/guards.py:2611] [34/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:05.853999 47307089929472 torch/_dynamo/guards.py:2611] [34/1] [__recompiles]     - tensor 'L['seq'][0]' size mismatch at index 0. expected 0, actual 4942
V0421 03:42:06.588111 47307089929472 torch/_dynamo/guards.py:2611] [20/1] [__recompiles] Recompiling function add_edges in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:435
V0421 03:42:06.588111 47307089929472 torch/_dynamo/guards.py:2611] [20/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:06.588111 47307089929472 torch/_dynamo/guards.py:2611] [20/1] [__recompiles]     - tensor 'L['u']' size mismatch at index 0. expected 4942, actual 970
V0421 03:42:06.608680 47307089929472 torch/_dynamo/guards.py:2611] [26/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_541 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:541
V0421 03:42:06.608680 47307089929472 torch/_dynamo/guards.py:2611] [26/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:06.608680 47307089929472 torch/_dynamo/guards.py:2611] [26/1] [__recompiles]     - tensor 'L['v']' size mismatch at index 0. expected 4942, actual 970
V0421 03:42:06.635640 47307089929472 torch/_dynamo/guards.py:2611] [27/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_542 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:542
V0421 03:42:06.635640 47307089929472 torch/_dynamo/guards.py:2611] [27/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:06.635640 47307089929472 torch/_dynamo/guards.py:2611] [27/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 4942, actual 970
V0421 03:42:06.651892 47307089929472 torch/_dynamo/guards.py:2611] [28/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_545 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:545
V0421 03:42:06.651892 47307089929472 torch/_dynamo/guards.py:2611] [28/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:06.651892 47307089929472 torch/_dynamo/guards.py:2611] [28/1] [__recompiles]     - tensor 'L['u']' size mismatch at index 0. expected 4942, actual 970
V0421 03:42:06.717236 47307089929472 torch/_dynamo/guards.py:2611] [30/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_573 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:573
V0421 03:42:06.717236 47307089929472 torch/_dynamo/guards.py:2611] [30/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:06.717236 47307089929472 torch/_dynamo/guards.py:2611] [30/1] [__recompiles]     - tensor 'L['u']' size mismatch at index 0. expected 4942, actual 970
V0421 03:42:06.961596 47307089929472 torch/_dynamo/guards.py:2611] [32/2] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_574 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:574
V0421 03:42:06.961596 47307089929472 torch/_dynamo/guards.py:2611] [32/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:06.961596 47307089929472 torch/_dynamo/guards.py:2611] [32/2] [__recompiles]     - tensor 'L['v']' size mismatch at index 0. expected 4942, actual 970
V0421 03:42:06.961596 47307089929472 torch/_dynamo/guards.py:2611] [32/2] [__recompiles]     - L['___stack0'] == 3459                                      
V0421 03:42:07.209141 47307089929472 torch/_dynamo/guards.py:2611] [34/2] [__recompiles] Recompiling function cat in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:222
V0421 03:42:07.209141 47307089929472 torch/_dynamo/guards.py:2611] [34/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:07.209141 47307089929472 torch/_dynamo/guards.py:2611] [34/2] [__recompiles]     - tensor 'L['seq'][1]' size mismatch at index 0. expected 4942, actual 970
V0421 03:42:07.209141 47307089929472 torch/_dynamo/guards.py:2611] [34/2] [__recompiles]     - tensor 'L['seq'][0]' size mismatch at index 0. expected 0, actual 9884
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/torch/_dynamo/variables/functions.py:663: UserWarning: Graph break due to unsupported builtin posix.stat. This function is either a Python builtin (e.g. _warnings.warn) or a third-party C/C++ Python extension (perhaps created with pybind). If it is a Python builtin, please file an issue on GitHub so the PyTorch team can add support for it and see the next case for a workaround. If it is a third-party C/C++ Python extension, please either wrap it into a PyTorch-understood custom operator (see https://pytorch.org/tutorials/advanced/custom_ops_landing_page.html for more details) or, if it is traceable, use torch.compiler.allow_in_graph.
  torch._dynamo.utils.warn_once(msg)
V0421 03:42:11.359974 47307089929472 torch/_dynamo/guards.py:2611] [6/3] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:432
V0421 03:42:11.359974 47307089929472 torch/_dynamo/guards.py:2611] [6/3] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:11.359974 47307089929472 torch/_dynamo/guards.py:2611] [6/3] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 03:42:11.359974 47307089929472 torch/_dynamo/guards.py:2611] [6/3] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 03:42:11.359974 47307089929472 torch/_dynamo/guards.py:2611] [6/3] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 03:42:11.406450 47307089929472 torch/_dynamo/guards.py:2611] [7/1] [__recompiles] Recompiling function torch_dynamo_resume_in_zerocopy_to_dgl_ndarray_at_435 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:435
V0421 03:42:11.406450 47307089929472 torch/_dynamo/guards.py:2611] [7/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:11.406450 47307089929472 torch/_dynamo/guards.py:2611] [7/1] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/torch/_dynamo/variables/functions.py:663: UserWarning: Graph break due to unsupported builtin time.perf_counter. This function is either a Python builtin (e.g. _warnings.warn) or a third-party C/C++ Python extension (perhaps created with pybind). If it is a Python builtin, please file an issue on GitHub so the PyTorch team can add support for it and see the next case for a workaround. If it is a third-party C/C++ Python extension, please either wrap it into a PyTorch-understood custom operator (see https://pytorch.org/tutorials/advanced/custom_ops_landing_page.html for more details) or, if it is traceable, use torch.compiler.allow_in_graph.
  torch._dynamo.utils.warn_once(msg)
V0421 03:42:11.649384 47307089929472 torch/_dynamo/guards.py:2611] [86/1] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 03:42:11.649384 47307089929472 torch/_dynamo/guards.py:2611] [86/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:11.649384 47307089929472 torch/_dynamo/guards.py:2611] [86/1] [__recompiles]     - L['___stack0'] == 928373.1215148                            
V0421 03:42:11.741815 47307089929472 torch/_dynamo/guards.py:2611] [86/2] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 03:42:11.741815 47307089929472 torch/_dynamo/guards.py:2611] [86/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:11.741815 47307089929472 torch/_dynamo/guards.py:2611] [86/2] [__recompiles]     - L['___stack0'] == 928373.13426266                           
V0421 03:42:11.741815 47307089929472 torch/_dynamo/guards.py:2611] [86/2] [__recompiles]     - L['___stack0'] == 928373.1215148                            
V0421 03:42:14.914761 47307089929472 torch/_dynamo/guards.py:2611] [86/3] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 03:42:14.914761 47307089929472 torch/_dynamo/guards.py:2611] [86/3] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:14.914761 47307089929472 torch/_dynamo/guards.py:2611] [86/3] [__recompiles]     - L['___stack0'] == 928373.226703785                          
V0421 03:42:14.914761 47307089929472 torch/_dynamo/guards.py:2611] [86/3] [__recompiles]     - L['___stack0'] == 928373.13426266                           
V0421 03:42:14.914761 47307089929472 torch/_dynamo/guards.py:2611] [86/3] [__recompiles]     - L['___stack0'] == 928373.1215148                            
V0421 03:42:16.850247 47307089929472 torch/_dynamo/guards.py:2611] [86/4] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 03:42:16.850247 47307089929472 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:16.850247 47307089929472 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     - L['___stack0'] == 928376.399323972                          
V0421 03:42:16.850247 47307089929472 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     - L['___stack0'] == 928373.226703785                          
V0421 03:42:16.850247 47307089929472 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     - L['___stack0'] == 928373.13426266                           
V0421 03:42:16.850247 47307089929472 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     - L['___stack0'] == 928373.1215148                            
V0421 03:42:16.919414 47307089929472 torch/_dynamo/guards.py:2611] [86/5] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 03:42:16.919414 47307089929472 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:16.919414 47307089929472 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 928378.33491463                           
V0421 03:42:16.919414 47307089929472 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 928376.399323972                          
V0421 03:42:16.919414 47307089929472 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 928373.226703785                          
V0421 03:42:16.919414 47307089929472 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 928373.13426266                           
V0421 03:42:16.919414 47307089929472 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 928373.1215148                            
V0421 03:42:16.925575 47307089929472 torch/_dynamo/guards.py:2611] [86/6] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 03:42:16.925575 47307089929472 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:16.925575 47307089929472 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 928378.404246483                          
V0421 03:42:16.925575 47307089929472 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 928378.33491463                           
V0421 03:42:16.925575 47307089929472 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 928376.399323972                          
V0421 03:42:16.925575 47307089929472 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 928373.226703785                          
V0421 03:42:16.925575 47307089929472 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 928373.13426266                           
V0421 03:42:16.925575 47307089929472 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 928373.1215148                            
V0421 03:42:16.983274 47307089929472 torch/_dynamo/guards.py:2611] [57/1] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/view.py:82
V0421 03:42:16.983274 47307089929472 torch/_dynamo/guards.py:2611] [57/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:16.983274 47307089929472 torch/_dynamo/guards.py:2611] [57/1] [__recompiles]     - tensor 'L['val']' requires_grad mismatch. expected requires_grad=0
V0421 03:42:17.005167 47307089929472 torch/_dynamo/guards.py:2611] [61/1] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:690
V0421 03:42:17.005167 47307089929472 torch/_dynamo/guards.py:2611] [61/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:17.005167 47307089929472 torch/_dynamo/guards.py:2611] [61/1] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 03:42:17.024237 47307089929472 torch/_dynamo/guards.py:2611] [62/1] [__recompiles] Recompiling function update_column in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:772
V0421 03:42:17.024237 47307089929472 torch/_dynamo/guards.py:2611] [62/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:17.024237 47307089929472 torch/_dynamo/guards.py:2611] [62/1] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 03:42:17.043664 47307089929472 torch/_dynamo/guards.py:2611] [63/1] [__recompiles] Recompiling function create in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:472
V0421 03:42:17.043664 47307089929472 torch/_dynamo/guards.py:2611] [63/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:17.043664 47307089929472 torch/_dynamo/guards.py:2611] [63/1] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 03:42:17.061737 47307089929472 torch/_dynamo/guards.py:2611] [64/1] [__recompiles] Recompiling function __init__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:210
V0421 03:42:17.061737 47307089929472 torch/_dynamo/guards.py:2611] [64/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:17.061737 47307089929472 torch/_dynamo/guards.py:2611] [64/1] [__recompiles]     - tensor 'L['storage']' requires_grad mismatch. expected requires_grad=0
V0421 03:42:17.080963 47307089929472 torch/_dynamo/guards.py:2611] [65/1] [__recompiles] Recompiling function infer_scheme in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:151
V0421 03:42:17.080963 47307089929472 torch/_dynamo/guards.py:2611] [65/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:17.080963 47307089929472 torch/_dynamo/guards.py:2611] [65/1] [__recompiles]     - tensor 'L['tensor']' requires_grad mismatch. expected requires_grad=0
V0421 03:42:17.125779 47307089929472 torch/_dynamo/guards.py:2611] [68/1] [__recompiles] Recompiling function func in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:213
V0421 03:42:17.125779 47307089929472 torch/_dynamo/guards.py:2611] [68/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:17.125779 47307089929472 torch/_dynamo/guards.py:2611] [68/1] [__recompiles]     - tensor 'L['x']' requires_grad mismatch. expected requires_grad=0
V0421 03:42:17.139188 47307089929472 torch/_dynamo/guards.py:2611] [70/1] [__recompiles] Recompiling function torch_dynamo_resume_in_gspmm_at_75 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:75
V0421 03:42:17.139188 47307089929472 torch/_dynamo/guards.py:2611] [70/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:17.139188 47307089929472 torch/_dynamo/guards.py:2611] [70/1] [__recompiles]     - tensor 'L['lhs_data']' requires_grad mismatch. expected requires_grad=0
V0421 03:42:17.156792 47307089929472 torch/_dynamo/guards.py:2611] [74/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/sparse.py:163
V0421 03:42:17.156792 47307089929472 torch/_dynamo/guards.py:2611] [74/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:17.156792 47307089929472 torch/_dynamo/guards.py:2611] [74/1] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=0
V0421 03:42:17.172821 47307089929472 torch/_dynamo/guards.py:2611] [77/1] [__recompiles] Recompiling function zeros in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:287
V0421 03:42:17.172821 47307089929472 torch/_dynamo/guards.py:2611] [77/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:17.172821 47307089929472 torch/_dynamo/guards.py:2611] [77/1] [__recompiles]     - L['shape'][1] == 767                                        
V0421 03:42:17.455762 47307089929472 torch/_dynamo/guards.py:2611] [6/4] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:432
V0421 03:42:17.455762 47307089929472 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:17.455762 47307089929472 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 03:42:17.455762 47307089929472 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 03:42:17.455762 47307089929472 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 03:42:17.455762 47307089929472 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0421 03:42:17.508861 47307089929472 torch/_dynamo/guards.py:2611] [79/1] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray_for_write in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:454
V0421 03:42:17.508861 47307089929472 torch/_dynamo/guards.py:2611] [79/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:17.508861 47307089929472 torch/_dynamo/guards.py:2611] [79/1] [__recompiles]     - tensor 'L['input']' stride mismatch at index 0. expected 767, actual 128
V0421 03:42:17.545358 47307089929472 torch/_dynamo/guards.py:2611] [80/1] [__recompiles] Recompiling function check_is_view in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:441
V0421 03:42:17.545358 47307089929472 torch/_dynamo/guards.py:2611] [80/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:17.545358 47307089929472 torch/_dynamo/guards.py:2611] [80/1] [__recompiles]     - tensor 'L['input']' stride mismatch at index 0. expected 767, actual 128
V0421 03:42:17.555299 47307089929472 torch/_dynamo/guards.py:2611] [81/1] [__recompiles] Recompiling function torch_dynamo_resume_in_check_is_view_at_443 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:443
V0421 03:42:17.555299 47307089929472 torch/_dynamo/guards.py:2611] [81/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:17.555299 47307089929472 torch/_dynamo/guards.py:2611] [81/1] [__recompiles]     - tensor 'L['input']' stride mismatch at index 0. expected 767, actual 128
V0421 03:42:17.566609 47307089929472 torch/_dynamo/guards.py:2611] [82/1] [__recompiles] Recompiling function torch_dynamo_resume_in_zerocopy_to_dgl_ndarray_for_write_at_461 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:461
V0421 03:42:17.566609 47307089929472 torch/_dynamo/guards.py:2611] [82/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:17.566609 47307089929472 torch/_dynamo/guards.py:2611] [82/1] [__recompiles]     - tensor 'L['input']' stride mismatch at index 0. expected 767, actual 128
V0421 03:42:17.581329 47307089929472 torch/_dynamo/guards.py:2611] [83/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_165 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/sparse.py:165
V0421 03:42:17.581329 47307089929472 torch/_dynamo/guards.py:2611] [83/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:17.581329 47307089929472 torch/_dynamo/guards.py:2611] [83/1] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=0
V0421 03:42:17.620424 47307089929472 torch/_dynamo/guards.py:2611] [86/7] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 03:42:17.620424 47307089929472 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:17.620424 47307089929472 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 928378.410420207                          
V0421 03:42:17.620424 47307089929472 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 928378.404246483                          
V0421 03:42:17.620424 47307089929472 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 928378.33491463                           
V0421 03:42:17.620424 47307089929472 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 928376.399323972                          
V0421 03:42:17.620424 47307089929472 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 928373.226703785                          
V0421 03:42:17.620424 47307089929472 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 928373.13426266                           
V0421 03:42:17.620424 47307089929472 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 928373.1215148                            
V0421 03:42:17.626850 47307089929472 torch/_dynamo/guards.py:2611] [86/8] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_12 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12
V0421 03:42:17.626850 47307089929472 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:17.626850 47307089929472 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 928379.105218852                          
V0421 03:42:17.626850 47307089929472 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 928378.410420207                          
V0421 03:42:17.626850 47307089929472 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 928378.404246483                          
V0421 03:42:17.626850 47307089929472 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 928378.33491463                           
V0421 03:42:17.626850 47307089929472 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 928376.399323972                          
V0421 03:42:17.626850 47307089929472 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 928373.226703785                          
V0421 03:42:17.626850 47307089929472 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 928373.13426266                           
V0421 03:42:17.626850 47307089929472 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 928373.1215148                            
W0421 03:42:17.626996 47307089929472 torch/_dynamo/convert_frame.py:762] [86/8] torch._dynamo hit config.cache_size_limit (8)
W0421 03:42:17.626996 47307089929472 torch/_dynamo/convert_frame.py:762] [86/8]    function: 'torch_dynamo_resume_in_hook_at_12' (/work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:12)
W0421 03:42:17.626996 47307089929472 torch/_dynamo/convert_frame.py:762] [86/8]    last reason: L['___stack0'] == 928373.1215148                            
W0421 03:42:17.626996 47307089929472 torch/_dynamo/convert_frame.py:762] [86/8] To log all recompilation reasons, use TORCH_LOGS="recompiles".
W0421 03:42:17.626996 47307089929472 torch/_dynamo/convert_frame.py:762] [86/8] To diagnose recompilation issues, see https://pytorch.org/docs/main/torch.compiler_troubleshooting.html.
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 03:42:19.445062 47307089929472 torch/_dynamo/guards.py:2611] [0/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/VQGraph/models.py:449
V0421 03:42:19.445062 47307089929472 torch/_dynamo/guards.py:2611] [0/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:19.445062 47307089929472 torch/_dynamo/guards.py:2611] [0/1] [__recompiles]     - tensor 'L['feats']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:19.475983 47307089929472 torch/_dynamo/guards.py:2611] [1/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/VQGraph/models.py:154
V0421 03:42:19.475983 47307089929472 torch/_dynamo/guards.py:2611] [1/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:19.475983 47307089929472 torch/_dynamo/guards.py:2611] [1/1] [__recompiles]     - tensor 'L['feats']' size mismatch at index 0. expected 3460, actual 3456
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 03:42:19.491810 47307089929472 torch/_dynamo/guards.py:2611] [9/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_157 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/models.py:157
V0421 03:42:19.491810 47307089929472 torch/_dynamo/guards.py:2611] [9/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:19.491810 47307089929472 torch/_dynamo/guards.py:2611] [9/1] [__recompiles]     - tensor 'L['h']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:19.506801 47307089929472 torch/_dynamo/guards.py:2611] [13/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_157 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/models.py:157
V0421 03:42:19.506801 47307089929472 torch/_dynamo/guards.py:2611] [13/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:19.506801 47307089929472 torch/_dynamo/guards.py:2611] [13/1] [__recompiles]     - tensor 'L['h']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:19.548794 47307089929472 torch/_dynamo/guards.py:2611] [4/3] [__recompiles] Recompiling function create_unitgraph_from_coo in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1254
V0421 03:42:19.548794 47307089929472 torch/_dynamo/guards.py:2611] [4/3] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:19.548794 47307089929472 torch/_dynamo/guards.py:2611] [4/3] [__recompiles]     - tensor 'L['row']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 03:42:19.548794 47307089929472 torch/_dynamo/guards.py:2611] [4/3] [__recompiles]     - L['num_dst'] == 3460                                        
V0421 03:42:19.548794 47307089929472 torch/_dynamo/guards.py:2611] [4/3] [__recompiles]     - L['num_dst'] == 3460                                        
V0421 03:42:19.581175 47307089929472 torch/_dynamo/guards.py:2611] [8/3] [__recompiles] Recompiling function torch_dynamo_resume_in_create_unitgraph_from_coo_at_1296 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1296
V0421 03:42:19.581175 47307089929472 torch/_dynamo/guards.py:2611] [8/3] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:19.581175 47307089929472 torch/_dynamo/guards.py:2611] [8/3] [__recompiles]     - tensor 'L['col']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 03:42:19.581175 47307089929472 torch/_dynamo/guards.py:2611] [8/3] [__recompiles]     - L['___stack2'] == 3460                                      
V0421 03:42:19.581175 47307089929472 torch/_dynamo/guards.py:2611] [8/3] [__recompiles]     - L['___stack2'] == 3460                                      
V0421 03:42:19.595254 47307089929472 torch/_dynamo/guards.py:2611] [15/1] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 03:42:19.595254 47307089929472 torch/_dynamo/guards.py:2611] [15/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:19.595254 47307089929472 torch/_dynamo/guards.py:2611] [15/1] [__recompiles]     - L['data'][0] == 3460                                        
V0421 03:42:22.887291 47307089929472 torch/_dynamo/guards.py:2611] [29/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_572 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:572
V0421 03:42:22.887291 47307089929472 torch/_dynamo/guards.py:2611] [29/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:22.887291 47307089929472 torch/_dynamo/guards.py:2611] [29/1] [__recompiles]     - L['___stack0'] == 3460                                      
V0421 03:42:22.905302 47307089929472 torch/_dynamo/guards.py:2611] [30/2] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_573 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:573
V0421 03:42:22.905302 47307089929472 torch/_dynamo/guards.py:2611] [30/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:22.905302 47307089929472 torch/_dynamo/guards.py:2611] [30/2] [__recompiles]     - L['___stack0'] == 3460                                      
V0421 03:42:22.905302 47307089929472 torch/_dynamo/guards.py:2611] [30/2] [__recompiles]     - tensor 'L['u']' size mismatch at index 0. expected 4942, actual 4848
V0421 03:42:23.159894 47307089929472 torch/_dynamo/guards.py:2611] [34/3] [__recompiles] Recompiling function cat in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:222
V0421 03:42:23.159894 47307089929472 torch/_dynamo/guards.py:2611] [34/3] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:23.159894 47307089929472 torch/_dynamo/guards.py:2611] [34/3] [__recompiles]     - 2 <= L['seq'][0].size()[0] <= 2147483645                      # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:42:23.159894 47307089929472 torch/_dynamo/guards.py:2611] [34/3] [__recompiles]     - tensor 'L['seq'][1]' size mismatch at index 0. expected 4942, actual 4848
V0421 03:42:23.159894 47307089929472 torch/_dynamo/guards.py:2611] [34/3] [__recompiles]     - tensor 'L['seq'][1]' size mismatch at index 0. expected 4942, actual 4848
V0421 03:42:23.389858 47307089929472 torch/_dynamo/guards.py:2611] [4/4] [__recompiles] Recompiling function create_unitgraph_from_coo in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1254
V0421 03:42:23.389858 47307089929472 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:23.389858 47307089929472 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     - tensor 'L['row']' size mismatch at index 0. expected 0, actual 4848
V0421 03:42:23.389858 47307089929472 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     - tensor 'L['row']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 03:42:23.389858 47307089929472 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     - L['num_dst'] == 3460                                        
V0421 03:42:23.389858 47307089929472 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     - tensor 'L['row']' size mismatch at index 0. expected 0, actual 4848
V0421 03:42:23.429737 47307089929472 torch/_dynamo/guards.py:2611] [8/4] [__recompiles] Recompiling function torch_dynamo_resume_in_create_unitgraph_from_coo_at_1296 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1296
V0421 03:42:23.429737 47307089929472 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:23.429737 47307089929472 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     - tensor 'L['col']' size mismatch at index 0. expected 0, actual 4848
V0421 03:42:23.429737 47307089929472 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     - tensor 'L['col']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0421 03:42:23.429737 47307089929472 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     - L['___stack2'] == 3460                                      
V0421 03:42:23.429737 47307089929472 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     - tensor 'L['col']' size mismatch at index 0. expected 0, actual 4848
V0421 03:42:23.449985 47307089929472 torch/_dynamo/guards.py:2611] [41/1] [__recompiles] Recompiling function spmatrix in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/sparse/sparse_matrix.py:763
V0421 03:42:23.449985 47307089929472 torch/_dynamo/guards.py:2611] [41/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:23.449985 47307089929472 torch/_dynamo/guards.py:2611] [41/1] [__recompiles]     - tensor 'L['indices']' stride mismatch at index 0. expected 11824, actual 11636
V0421 03:42:23.720049 47307089929472 torch/_dynamo/guards.py:2611] [47/1] [__recompiles] Recompiling function arange in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:406
V0421 03:42:23.720049 47307089929472 torch/_dynamo/guards.py:2611] [47/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:23.720049 47307089929472 torch/_dynamo/guards.py:2611] [47/1] [__recompiles]     - L['stop'] == 3460                                           
V0421 03:42:23.941451 47307089929472 torch/_dynamo/guards.py:2611] [48/1] [__recompiles] Recompiling function in_degrees in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:702
V0421 03:42:23.941451 47307089929472 torch/_dynamo/guards.py:2611] [48/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:23.941451 47307089929472 torch/_dynamo/guards.py:2611] [48/1] [__recompiles]     - tensor 'L['v']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:23.959112 47307089929472 torch/_dynamo/guards.py:2611] [50/1] [__recompiles] Recompiling function has_nodes in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:2878
V0421 03:42:23.959112 47307089929472 torch/_dynamo/guards.py:2611] [50/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:23.959112 47307089929472 torch/_dynamo/guards.py:2611] [50/1] [__recompiles]     - tensor 'L['vid']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:23.975001 47307089929472 torch/_dynamo/guards.py:2611] [51/1] [__recompiles] Recompiling function torch_dynamo_resume_in_has_nodes_at_2925 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:2925
V0421 03:42:23.975001 47307089929472 torch/_dynamo/guards.py:2611] [51/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:23.975001 47307089929472 torch/_dynamo/guards.py:2611] [51/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:24.212772 47307089929472 torch/_dynamo/guards.py:2611] [52/1] [__recompiles] Recompiling function torch_dynamo_resume_in_has_nodes_at_2926 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:2926
V0421 03:42:24.212772 47307089929472 torch/_dynamo/guards.py:2611] [52/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:24.212772 47307089929472 torch/_dynamo/guards.py:2611] [52/1] [__recompiles]     - tensor 'L['vid_tensor']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:24.235761 47307089929472 torch/_dynamo/guards.py:2611] [53/1] [__recompiles] Recompiling function has_nodes in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:444
V0421 03:42:24.235761 47307089929472 torch/_dynamo/guards.py:2611] [53/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:24.235761 47307089929472 torch/_dynamo/guards.py:2611] [53/1] [__recompiles]     - tensor 'L['vids']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:24.253250 47307089929472 torch/_dynamo/guards.py:2611] [54/1] [__recompiles] Recompiling function torch_dynamo_resume_in_has_nodes_at_2930 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:2930
V0421 03:42:24.253250 47307089929472 torch/_dynamo/guards.py:2611] [54/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:24.253250 47307089929472 torch/_dynamo/guards.py:2611] [54/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:24.472653 47307089929472 torch/_dynamo/guards.py:2611] [55/1] [__recompiles] Recompiling function sum in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:152
V0421 03:42:24.472653 47307089929472 torch/_dynamo/guards.py:2611] [55/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:24.472653 47307089929472 torch/_dynamo/guards.py:2611] [55/1] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:24.551618 47307089929472 torch/_dynamo/guards.py:2611] [57/2] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/view.py:82
V0421 03:42:24.551618 47307089929472 torch/_dynamo/guards.py:2611] [57/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:24.551618 47307089929472 torch/_dynamo/guards.py:2611] [57/2] [__recompiles]     - tensor 'L['val']' requires_grad mismatch. expected requires_grad=1
V0421 03:42:24.551618 47307089929472 torch/_dynamo/guards.py:2611] [57/2] [__recompiles]     - tensor 'L['val']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:24.569209 47307089929472 torch/_dynamo/guards.py:2611] [61/2] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:690
V0421 03:42:24.569209 47307089929472 torch/_dynamo/guards.py:2611] [61/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:24.569209 47307089929472 torch/_dynamo/guards.py:2611] [61/2] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=1
V0421 03:42:24.569209 47307089929472 torch/_dynamo/guards.py:2611] [61/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:24.587166 47307089929472 torch/_dynamo/guards.py:2611] [62/2] [__recompiles] Recompiling function update_column in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:772
V0421 03:42:24.587166 47307089929472 torch/_dynamo/guards.py:2611] [62/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:24.587166 47307089929472 torch/_dynamo/guards.py:2611] [62/2] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=1
V0421 03:42:24.587166 47307089929472 torch/_dynamo/guards.py:2611] [62/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:24.606687 47307089929472 torch/_dynamo/guards.py:2611] [63/2] [__recompiles] Recompiling function create in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:472
V0421 03:42:24.606687 47307089929472 torch/_dynamo/guards.py:2611] [63/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:24.606687 47307089929472 torch/_dynamo/guards.py:2611] [63/2] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=1
V0421 03:42:24.606687 47307089929472 torch/_dynamo/guards.py:2611] [63/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:24.622932 47307089929472 torch/_dynamo/guards.py:2611] [64/2] [__recompiles] Recompiling function __init__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:210
V0421 03:42:24.622932 47307089929472 torch/_dynamo/guards.py:2611] [64/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:24.622932 47307089929472 torch/_dynamo/guards.py:2611] [64/2] [__recompiles]     - tensor 'L['storage']' requires_grad mismatch. expected requires_grad=1
V0421 03:42:24.622932 47307089929472 torch/_dynamo/guards.py:2611] [64/2] [__recompiles]     - tensor 'L['storage']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:24.641309 47307089929472 torch/_dynamo/guards.py:2611] [65/2] [__recompiles] Recompiling function infer_scheme in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:151
V0421 03:42:24.641309 47307089929472 torch/_dynamo/guards.py:2611] [65/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:24.641309 47307089929472 torch/_dynamo/guards.py:2611] [65/2] [__recompiles]     - tensor 'L['tensor']' requires_grad mismatch. expected requires_grad=1
V0421 03:42:24.641309 47307089929472 torch/_dynamo/guards.py:2611] [65/2] [__recompiles]     - tensor 'L['tensor']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:24.683846 47307089929472 torch/_dynamo/guards.py:2611] [68/2] [__recompiles] Recompiling function func in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:213
V0421 03:42:24.683846 47307089929472 torch/_dynamo/guards.py:2611] [68/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:24.683846 47307089929472 torch/_dynamo/guards.py:2611] [68/2] [__recompiles]     - tensor 'L['x']' requires_grad mismatch. expected requires_grad=1
V0421 03:42:24.683846 47307089929472 torch/_dynamo/guards.py:2611] [68/2] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:24.697125 47307089929472 torch/_dynamo/guards.py:2611] [70/2] [__recompiles] Recompiling function torch_dynamo_resume_in_gspmm_at_75 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:75
V0421 03:42:24.697125 47307089929472 torch/_dynamo/guards.py:2611] [70/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:24.697125 47307089929472 torch/_dynamo/guards.py:2611] [70/2] [__recompiles]     - tensor 'L['lhs_data']' requires_grad mismatch. expected requires_grad=1
V0421 03:42:24.697125 47307089929472 torch/_dynamo/guards.py:2611] [70/2] [__recompiles]     - tensor 'L['lhs_data']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:24.715297 47307089929472 torch/_dynamo/guards.py:2611] [74/2] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/sparse.py:163
V0421 03:42:24.715297 47307089929472 torch/_dynamo/guards.py:2611] [74/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:24.715297 47307089929472 torch/_dynamo/guards.py:2611] [74/2] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=1
V0421 03:42:24.715297 47307089929472 torch/_dynamo/guards.py:2611] [74/2] [__recompiles]     - tensor 'L['X']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:24.731218 47307089929472 torch/_dynamo/guards.py:2611] [77/2] [__recompiles] Recompiling function zeros in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:287
V0421 03:42:24.731218 47307089929472 torch/_dynamo/guards.py:2611] [77/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:24.731218 47307089929472 torch/_dynamo/guards.py:2611] [77/2] [__recompiles]     - L['shape'][0] == 3460                                       
V0421 03:42:24.731218 47307089929472 torch/_dynamo/guards.py:2611] [77/2] [__recompiles]     - L['shape'][0] == 3460                                       
V0421 03:42:24.890659 47307089929472 torch/_dynamo/guards.py:2611] [79/2] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray_for_write in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:454
V0421 03:42:24.890659 47307089929472 torch/_dynamo/guards.py:2611] [79/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:24.890659 47307089929472 torch/_dynamo/guards.py:2611] [79/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:24.890659 47307089929472 torch/_dynamo/guards.py:2611] [79/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:24.930904 47307089929472 torch/_dynamo/guards.py:2611] [80/2] [__recompiles] Recompiling function check_is_view in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:441
V0421 03:42:24.930904 47307089929472 torch/_dynamo/guards.py:2611] [80/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:24.930904 47307089929472 torch/_dynamo/guards.py:2611] [80/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:24.930904 47307089929472 torch/_dynamo/guards.py:2611] [80/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:24.942176 47307089929472 torch/_dynamo/guards.py:2611] [81/2] [__recompiles] Recompiling function torch_dynamo_resume_in_check_is_view_at_443 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:443
V0421 03:42:24.942176 47307089929472 torch/_dynamo/guards.py:2611] [81/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:24.942176 47307089929472 torch/_dynamo/guards.py:2611] [81/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:24.942176 47307089929472 torch/_dynamo/guards.py:2611] [81/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:24.955647 47307089929472 torch/_dynamo/guards.py:2611] [82/2] [__recompiles] Recompiling function torch_dynamo_resume_in_zerocopy_to_dgl_ndarray_for_write_at_461 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:461
V0421 03:42:24.955647 47307089929472 torch/_dynamo/guards.py:2611] [82/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:24.955647 47307089929472 torch/_dynamo/guards.py:2611] [82/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:24.955647 47307089929472 torch/_dynamo/guards.py:2611] [82/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:24.971827 47307089929472 torch/_dynamo/guards.py:2611] [88/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:569
V0421 03:42:24.971827 47307089929472 torch/_dynamo/guards.py:2611] [88/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:24.971827 47307089929472 torch/_dynamo/guards.py:2611] [88/1] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:25.061700 47307089929472 torch/_dynamo/guards.py:2611] [89/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_590 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:590
V0421 03:42:25.061700 47307089929472 torch/_dynamo/guards.py:2611] [89/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:25.061700 47307089929472 torch/_dynamo/guards.py:2611] [89/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:25.122818 47307089929472 torch/_dynamo/guards.py:2611] [90/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:418
V0421 03:42:25.122818 47307089929472 torch/_dynamo/guards.py:2611] [90/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:25.122818 47307089929472 torch/_dynamo/guards.py:2611] [90/1] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:25.694847 47307089929472 torch/_dynamo/guards.py:2611] [93/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_432 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:432
V0421 03:42:25.694847 47307089929472 torch/_dynamo/guards.py:2611] [93/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:25.694847 47307089929472 torch/_dynamo/guards.py:2611] [93/1] [__recompiles]     - L['shape'][1] == 3460                                       
V0421 03:42:27.012911 47307089929472 torch/_dynamo/guards.py:2611] [94/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_595 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:595
V0421 03:42:27.012911 47307089929472 torch/_dynamo/guards.py:2611] [94/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:27.012911 47307089929472 torch/_dynamo/guards.py:2611] [94/1] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:27.258827 47307089929472 torch/_dynamo/guards.py:2611] [95/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_604 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:604
V0421 03:42:27.258827 47307089929472 torch/_dynamo/guards.py:2611] [95/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:27.258827 47307089929472 torch/_dynamo/guards.py:2611] [95/1] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:27.967166 47307089929472 torch/_dynamo/guards.py:2611] [96/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_648 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:648
V0421 03:42:27.967166 47307089929472 torch/_dynamo/guards.py:2611] [96/1] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:27.967166 47307089929472 torch/_dynamo/guards.py:2611] [96/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:28.069656 47307089929472 torch/_dynamo/guards.py:2611] [57/3] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/view.py:82
V0421 03:42:28.069656 47307089929472 torch/_dynamo/guards.py:2611] [57/3] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:28.069656 47307089929472 torch/_dynamo/guards.py:2611] [57/3] [__recompiles]     - tensor 'L['val']' requires_grad mismatch. expected requires_grad=0
V0421 03:42:28.069656 47307089929472 torch/_dynamo/guards.py:2611] [57/3] [__recompiles]     - tensor 'L['val']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:28.069656 47307089929472 torch/_dynamo/guards.py:2611] [57/3] [__recompiles]     - tensor 'L['val']' requires_grad mismatch. expected requires_grad=0
V0421 03:42:28.089988 47307089929472 torch/_dynamo/guards.py:2611] [61/3] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:690
V0421 03:42:28.089988 47307089929472 torch/_dynamo/guards.py:2611] [61/3] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:28.089988 47307089929472 torch/_dynamo/guards.py:2611] [61/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 03:42:28.089988 47307089929472 torch/_dynamo/guards.py:2611] [61/3] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:28.089988 47307089929472 torch/_dynamo/guards.py:2611] [61/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 03:42:28.110400 47307089929472 torch/_dynamo/guards.py:2611] [62/3] [__recompiles] Recompiling function update_column in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:772
V0421 03:42:28.110400 47307089929472 torch/_dynamo/guards.py:2611] [62/3] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:28.110400 47307089929472 torch/_dynamo/guards.py:2611] [62/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 03:42:28.110400 47307089929472 torch/_dynamo/guards.py:2611] [62/3] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:28.110400 47307089929472 torch/_dynamo/guards.py:2611] [62/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 03:42:28.131100 47307089929472 torch/_dynamo/guards.py:2611] [63/3] [__recompiles] Recompiling function create in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:472
V0421 03:42:28.131100 47307089929472 torch/_dynamo/guards.py:2611] [63/3] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:28.131100 47307089929472 torch/_dynamo/guards.py:2611] [63/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 03:42:28.131100 47307089929472 torch/_dynamo/guards.py:2611] [63/3] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:28.131100 47307089929472 torch/_dynamo/guards.py:2611] [63/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0421 03:42:28.149735 47307089929472 torch/_dynamo/guards.py:2611] [64/3] [__recompiles] Recompiling function __init__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:210
V0421 03:42:28.149735 47307089929472 torch/_dynamo/guards.py:2611] [64/3] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:28.149735 47307089929472 torch/_dynamo/guards.py:2611] [64/3] [__recompiles]     - tensor 'L['storage']' requires_grad mismatch. expected requires_grad=0
V0421 03:42:28.149735 47307089929472 torch/_dynamo/guards.py:2611] [64/3] [__recompiles]     - tensor 'L['storage']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:28.149735 47307089929472 torch/_dynamo/guards.py:2611] [64/3] [__recompiles]     - tensor 'L['storage']' requires_grad mismatch. expected requires_grad=0
V0421 03:42:28.170411 47307089929472 torch/_dynamo/guards.py:2611] [65/3] [__recompiles] Recompiling function infer_scheme in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:151
V0421 03:42:28.170411 47307089929472 torch/_dynamo/guards.py:2611] [65/3] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:28.170411 47307089929472 torch/_dynamo/guards.py:2611] [65/3] [__recompiles]     - tensor 'L['tensor']' requires_grad mismatch. expected requires_grad=0
V0421 03:42:28.170411 47307089929472 torch/_dynamo/guards.py:2611] [65/3] [__recompiles]     - tensor 'L['tensor']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:28.170411 47307089929472 torch/_dynamo/guards.py:2611] [65/3] [__recompiles]     - tensor 'L['tensor']' requires_grad mismatch. expected requires_grad=0
V0421 03:42:28.218435 47307089929472 torch/_dynamo/guards.py:2611] [68/3] [__recompiles] Recompiling function func in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:213
V0421 03:42:28.218435 47307089929472 torch/_dynamo/guards.py:2611] [68/3] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:28.218435 47307089929472 torch/_dynamo/guards.py:2611] [68/3] [__recompiles]     - tensor 'L['x']' requires_grad mismatch. expected requires_grad=0
V0421 03:42:28.218435 47307089929472 torch/_dynamo/guards.py:2611] [68/3] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:28.218435 47307089929472 torch/_dynamo/guards.py:2611] [68/3] [__recompiles]     - tensor 'L['x']' requires_grad mismatch. expected requires_grad=0
V0421 03:42:28.232889 47307089929472 torch/_dynamo/guards.py:2611] [70/3] [__recompiles] Recompiling function torch_dynamo_resume_in_gspmm_at_75 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:75
V0421 03:42:28.232889 47307089929472 torch/_dynamo/guards.py:2611] [70/3] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:28.232889 47307089929472 torch/_dynamo/guards.py:2611] [70/3] [__recompiles]     - tensor 'L['lhs_data']' requires_grad mismatch. expected requires_grad=0
V0421 03:42:28.232889 47307089929472 torch/_dynamo/guards.py:2611] [70/3] [__recompiles]     - tensor 'L['lhs_data']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:28.232889 47307089929472 torch/_dynamo/guards.py:2611] [70/3] [__recompiles]     - tensor 'L['lhs_data']' requires_grad mismatch. expected requires_grad=0
V0421 03:42:28.251434 47307089929472 torch/_dynamo/guards.py:2611] [74/3] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/sparse.py:163
V0421 03:42:28.251434 47307089929472 torch/_dynamo/guards.py:2611] [74/3] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:28.251434 47307089929472 torch/_dynamo/guards.py:2611] [74/3] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=0
V0421 03:42:28.251434 47307089929472 torch/_dynamo/guards.py:2611] [74/3] [__recompiles]     - tensor 'L['X']' size mismatch at index 0. expected 3460, actual 3456
V0421 03:42:28.251434 47307089929472 torch/_dynamo/guards.py:2611] [74/3] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=0
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 03:42:29.299104 47307089929472 torch/_dynamo/guards.py:2611] [15/2] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 03:42:29.299104 47307089929472 torch/_dynamo/guards.py:2611] [15/2] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:29.299104 47307089929472 torch/_dynamo/guards.py:2611] [15/2] [__recompiles]     - L['data'][0] == 3456                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:42:29.299104 47307089929472 torch/_dynamo/guards.py:2611] [15/2] [__recompiles]     - L['data'][0] == 3460                                        
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 03:42:33.611874 47307089929472 torch/_dynamo/guards.py:2611] [15/3] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 03:42:33.611874 47307089929472 torch/_dynamo/guards.py:2611] [15/3] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:33.611874 47307089929472 torch/_dynamo/guards.py:2611] [15/3] [__recompiles]     - L['data'][0] == 3501                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:42:33.611874 47307089929472 torch/_dynamo/guards.py:2611] [15/3] [__recompiles]     - L['data'][0] == 3456                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:42:33.611874 47307089929472 torch/_dynamo/guards.py:2611] [15/3] [__recompiles]     - L['data'][0] == 3460                                        
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 03:42:37.925042 47307089929472 torch/_dynamo/guards.py:2611] [15/4] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 03:42:37.925042 47307089929472 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:37.925042 47307089929472 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     - L['data'][0] == 3443                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:42:37.925042 47307089929472 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     - L['data'][0] == 3501                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:42:37.925042 47307089929472 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     - L['data'][0] == 3456                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:42:37.925042 47307089929472 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     - L['data'][0] == 3460                                        
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 03:42:42.243683 47307089929472 torch/_dynamo/guards.py:2611] [15/5] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 03:42:42.243683 47307089929472 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:42.243683 47307089929472 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 3480                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:42:42.243683 47307089929472 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 3443                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:42:42.243683 47307089929472 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 3501                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:42:42.243683 47307089929472 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 3456                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:42:42.243683 47307089929472 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 3460                                        
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 03:42:46.563611 47307089929472 torch/_dynamo/guards.py:2611] [15/6] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 03:42:46.563611 47307089929472 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:46.563611 47307089929472 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 3462                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:42:46.563611 47307089929472 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 3480                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:42:46.563611 47307089929472 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 3443                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:42:46.563611 47307089929472 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 3501                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:42:46.563611 47307089929472 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 3456                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:42:46.563611 47307089929472 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 3460                                        
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 03:42:50.878638 47307089929472 torch/_dynamo/guards.py:2611] [15/7] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 03:42:50.878638 47307089929472 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:50.878638 47307089929472 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 3504                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:42:50.878638 47307089929472 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 3462                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:42:50.878638 47307089929472 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 3480                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:42:50.878638 47307089929472 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 3443                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:42:50.878638 47307089929472 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 3501                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:42:50.878638 47307089929472 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 3456                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:42:50.878638 47307089929472 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 3460                                        
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0421 03:42:55.174350 47307089929472 torch/_dynamo/guards.py:2611] [15/8] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0421 03:42:55.174350 47307089929472 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     triggered by the following guard failure(s):
V0421 03:42:55.174350 47307089929472 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 3450                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:42:55.174350 47307089929472 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 3504                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:42:55.174350 47307089929472 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 3462                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:42:55.174350 47307089929472 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 3480                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:42:55.174350 47307089929472 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 3443                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:42:55.174350 47307089929472 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 3501                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:42:55.174350 47307089929472 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 3456                                          # _dynamo/output_graph.py:452 in init_ambient_guards
V0421 03:42:55.174350 47307089929472 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 3460                                        
W0421 03:42:55.174586 47307089929472 torch/_dynamo/convert_frame.py:762] [15/8] torch._dynamo hit config.cache_size_limit (8)
W0421 03:42:55.174586 47307089929472 torch/_dynamo/convert_frame.py:762] [15/8]    function: '_dispatch' (/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65)
W0421 03:42:55.174586 47307089929472 torch/_dynamo/convert_frame.py:762] [15/8]    last reason: L['data'][0] == 3460                                        
W0421 03:42:55.174586 47307089929472 torch/_dynamo/convert_frame.py:762] [15/8] To log all recompilation reasons, use TORCH_LOGS="recompiles".
W0421 03:42:55.174586 47307089929472 torch/_dynamo/convert_frame.py:762] [15/8] To diagnose recompilation issues, see https://pytorch.org/docs/main/torch.compiler_troubleshooting.html.
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
Compiling model with torch.compile()...
Saved sparsity snapshot to outputs_compiled/tran/a-computer/SAGE/seed_2/sparsity_before.csv
                               layer_name  num_elements   sparsity
0  _orig_mod.encoder.graph_layer_1.weight        588289    0.00000
1    _orig_mod.encoder.graph_layer_1.bias           767  100.00000
2  _orig_mod.encoder.graph_layer_2.weight         98176    0.00000
3    _orig_mod.encoder.graph_layer_2.bias           128  100.00000
4      _orig_mod.encoder.decoder_1.weight        588289    0.00000
5        _orig_mod.encoder.decoder_1.bias           767    0.00000
6      _orig_mod.encoder.decoder_2.weight        588289    0.00017
7        _orig_mod.encoder.decoder_2.bias           767    0.00000
8         _orig_mod.encoder.linear.weight          1280    0.00000
9           _orig_mod.encoder.linear.bias            10    0.00000
Saved sparsity snapshot to outputs_compiled/tran/a-computer/SAGE/seed_2/sparsity_after.csv
                               layer_name  num_elements   sparsity
0  _orig_mod.encoder.graph_layer_1.weight        588289   0.222680
1    _orig_mod.encoder.graph_layer_1.bias           767   0.000000
2  _orig_mod.encoder.graph_layer_2.weight         98176   0.467528
3    _orig_mod.encoder.graph_layer_2.bias           128   0.000000
4      _orig_mod.encoder.decoder_1.weight        588289   5.088315
5        _orig_mod.encoder.decoder_1.bias           767   0.000000
6      _orig_mod.encoder.decoder_2.weight        588289  35.246282
7        _orig_mod.encoder.decoder_2.bias           767  12.125163
8         _orig_mod.encoder.linear.weight          1280   0.000000
9           _orig_mod.encoder.linear.bias            10   0.000000
Saved forward timing report to outputs_compiled/tran/a-computer/SAGE/seed_2/forward_times.csv
                         layer_name  avg_forward_time_ms  num_calls
0         _orig_mod.encoder.dropout             0.000605      14027
1   _orig_mod.encoder.graph_layer_1             0.000877      14027
2   _orig_mod.encoder.graph_layer_2             0.000748      14027
3       _orig_mod.encoder.decoder_1             0.000680      14027
4       _orig_mod.encoder.decoder_2             0.000661      14027
5          _orig_mod.encoder.linear             0.000290      14027
6   _orig_mod.encoder.vq.project_in             0.000595      14027
7  _orig_mod.encoder.vq.project_out             0.000754      14027
8    _orig_mod.encoder.vq._codebook             0.000879      14027
CPU Memory Usage: 1301.41 MB
Max GPU Memory Used: 2082.01 MB
